client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -18,47 +18,67 @@ import HeaderCake from 'components/header-cake';
 import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
 import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
 import SectionHeader from 'components/section-header';
-import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { addZone } from '../../../state/zones/actions';
 
 const form = 'extensions.zoninator.newZone';
 
-const ZoneCreator = ( { siteSlug, translate } ) => (
-	<div>
-		<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` }>
-			{ translate( 'Add a zone' ) }
-		</HeaderCake>
+const ZoneCreator = ( {
+	handleSubmit,
+	isDisabled,
+	saveZone,
+	siteId,
+	siteSlug,
+	submitting,
+	translate,
+} ) => {
+	const save = data => saveZone( siteId, form, data );
 
-		<form>
-			<SectionHeader label={ translate( 'New zone' ) }>
-				<FormButton compact>{ translate( 'Save' ) }</FormButton>
-			</SectionHeader>
-			<CompactCard>
-				<FormFieldset>
-					<FormLabel htmlFor="zoneName">{ translate( 'Zone name' ) }</FormLabel>
-					<ReduxFormTextInput name="zoneName" />
-				</FormFieldset>
+	return (
+		<div>
+			<HeaderCake backHref={ `/extensions/zoninator/${ siteSlug }` }>
+				{ translate( 'Add a zone' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **14** times:
`translate( 'Add a DNS Zone' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/zoninator-new-zone-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).